src/components/Profile/Interests/styles.scss

```diff
@@ -0,0 +1,36 @@
+.interests-header {
+    font-size: 22px;
+    font-weight: 500;
+    padding: 0 0.75rem;
+    color: #000c4d;
+}
+
+.interests-button {
+    width: 258px;
+    text-transform: uppercase;
+    border-radius: 24.5px;
+    border: solid 1px #f5a623;
+    background-color: #fff2e7;
+
+    white-space: nowrap;
+    vertical-align: middle;
+    user-select: none;
+    padding: 0 0.75rem;
+    font-size: 0.875rem;
+    height: 36px;
+    -webkit-transition: all 0.15s ease-in-out;
+    transition: all 0.15s ease-in-out;
+    color: #1d1d1d;
+    cursor: pointer;
+    outline: none;
+}
+
+.interests-text {
+    padding: 0 0.75rem;
+    margin-top: 20px;
+    margin-bottom: 40px;
+}
+
+.centered-div {
```

Comment:
CSS classes should describe what some component is: a class could be e.g. `title`, `title icon`, `description`, `box-header` etc